src/Helper/HeadScript.php

```diff
@@ -360,10 +360,11 @@ public function captureEnd()
      * Create data item containing all necessary components of script
      *
      * @internal This method will become private in version 3.0
+     * @deprecated Since 2.40.0. This method will bew removed in 3.0
```

Comment:
The line with `@internal` is now obsolete, correct?

Reply:
Also missed a couple of constants here